### PR TITLE
changes for next android release

### PR DESCRIPTION
**Pull request by jeaninesw**
Changes resulting from `fastlane android beta`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
     "@react-native-community/push-notification-ios": "^1.2.2",
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.5",
-    "@t2tx/react-native-universal-pedometer": "https://github.com/francisli/react-native-universal-pedometer#b50b4766de3bc487147d7c314fe36abc7763e730",
```
Comment (francisli) [inline package.json:18]: This package is used by the iOS app for the pedometer... was it required for you to remove it in order to get the build to succeed?

**Review by francisli — COMMENTED** [review]
Just one question about the removed node package, removing it will break the iOS app...

Did you use the fastlane script to release the build? If the upload was successful, it should have automatically committed the version change, tagged the commit, and pushed everything up...

File: package.json (hunk 1 above)
Comment (jeaninesw) [reply package.json:18]: I didn't manually remove this package, so I'm assuming it happened from my `fastlane android beta` command. I couldn't tell you why it didn't push everything up, but it did make a local commit. That commit had some extra garbage so I cleaned it up.

Comment (jeaninesw) [reply package.json:18]: If you think it'd be better, I could update the PR to just be `build.gradle`.

Comment (francisli) [reply package.json:18]: Yes, let's revert the package.json and package-lock.json changes...

Comment (jeaninesw) [reply package.json]: Done

**Pull request merged**